docs/how-to-guides/tracing/index.md

```diff
@@ -4,11 +4,13 @@
 This is an experimental feature, and may change at any time. Learn [more](../faq.md#stable-vs-experimental).
 :::
 
-Prompt flow provides the trace feature to capture and visualize the internal execution details for all flows.
+Prompt flow provides the trace feature to enable user to trace LLM call or function, and LLM frameworks like `LangChain` and `AutoGen`, following [OpenTelemetry specification](https://opentelemetry.io/docs/specs/otel/).
```

Comment:
I feel like the introductin to trace is too brief, we are assuming the reader is a machine learning veteran. For a normal developer that is not that familiar with trace. I would like to know something like:
- What does tracing mean
- What metrics/data will it record.
- Why do I need it?
- etc..

Reply:
good point, let me add an overview for traces

Reply:
updated some with GPT, may take a look

Reply:
much better now.